pydra/utils/tests/test_typing.py

```diff
@@ -194,6 +207,12 @@ def test_type_check_fail2():
         TypeParser(ty.Union[Path, File])(lz(int))
 
 
+@pytest.mark.skipif(sys.version_info < (3, 10), reason="No UnionType < Py3.10")
+def test_type_check_fail2a():
+    with pytest.raises(TypeError, match="to any of the union types"):
+        TypeParser(Path | File)(lz(int))
```

Comment:
Isn't it possible to run this test with a `from __future__ import annotations` for Python < 3.10?

Reply:
`from __future__ import annotations` just leaves all annotations as strings I believe. Probably need to have a separate test module to handle that case, but since it doesn't look like that will be the way that types are handled going forward I have tended to avoid it